app/institutions/dashboard/-components/institutional-users-list/component.ts

```diff
@@ -0,0 +1,63 @@
+import { action, computed } from '@ember-decorators/object';
+import { alias } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import InstitutionModel from 'ember-osf-web/models/institution';
+
+export default class InstitutionalUsersList extends Component {
+    @alias('model.taskInstance.value') institution!: InstitutionModel;
+
+    // Private properties
+    @service i18n!: I18N;
+    department?: string;
+    sort = 'user_full_name';
+
+    @computed('i18n.locale')
+    get defaultDepartment() {
+        return this.i18n.t('institutions.dashboard.select_default');
+    }
+
+    @computed('institution')
+    get departments() {
+        let departments = [this.defaultDepartment];
+
+        if (this.institution) {
+            const institutionDepartments = this.institution.statSummary.departments.map((x: any) => x.name);
+            departments = departments.concat(institutionDepartments);
+        }
+
+        if (!this.department) {
+            this.set('department', departments[0]);
+        }
+
+        return departments;
+    }
+
+    @computed('department', 'departments')
```

Comment:
This doesn't depend on `departments`, but *does* depend on `defaultDepartment`, which could change when the locale changes. (we've talked about allowing the user to change locales on the fly)